lang/en.json

```diff
@@ -1071,6 +1071,10 @@
         "ClickHtmlAttributeAttributeDescription": "The name of the attribute, such as title, value, alt, or any other HTML attribute.",
         "ClickHtmlAttributeAttributePlaceholder": "e.g. value",
         "PausedTag": "Fantastic! The tag has been paused successfully.",
-        "ResumedTag": "Fantastic! The tag has been resumed successfully."
+        "ResumedTag": "Fantastic! The tag has been resumed successfully.",
+        "MatomoConfigurationMatomoTrackBotsDescription": "By default, Matomo doesn't track visits by bots in order to show accurate visitor metrics. Enable this feature to add tracking for bot visits.",
```

Comment:
```suggestion
```

Seems that one was duplicated by accident

Reply:
Thanks, was added by mistakenly, removed :+1: